TeX.gitignore

```diff
@@ -111,9 +111,12 @@ acs-*.bib
 *.glsdefs
 *.lzo
 *.lzs
+*.slg
+*.slo
+*.sls
 
 # uncomment this for glossaries-extra (will ignore makeindex's style files!)
-# *.ist
+*.ist
```

Comment:
Do we need to enable this rule by default? The comment above indicates it'll now break another extension, so I want to understand this change a bit better and ensure it's correctly documented...